dev-docs/monitoring-suggestions.md

```diff
@@ -0,0 +1,46 @@
+# Monitoring Suggestions
+
+Taskcluster has several background processes that you should ensure are running on a schedule. Any of the following will generate messages
+of the form:
+
+```json
+{
+  "Type": "monitor.periodic",
+  "Logger": "<Logger>",
+  "Fields": {
+    "name": "<Name>"
+  }
+}
+```
+
+They will also have Fields for `status`, `duration`, and a serialized `error` if an error occured.
+
+The processes that have `continuous` for their dedaline and schedule run every few minutes and should complete fairly quickly. The rest
+have their schedules and maximum allowed duration defined here.
+
+| Service        | Name                   | Logger                     | Deadline (seconds) | Schedule    |
+| -------------- | ---------------------- | -------------------------- | ------------------ | ----------- |
+| auth           | purgeExpiredClients    | taskcluster.auth           | 86400              | At 12:00 AM |
```

Comment:
Should the schedule say "UTC"?

Reply:
I guess it depends on what k8s thinks about timezones and cron. I'll look that up shortly.

Reply:
Everything I've read so far says it is just based on the timezone of the master.

Reply:
Maybe a sentence to that effect?